src/opower/utilities/exelon.py

```diff
@@ -156,6 +156,7 @@ async def async_login(
             ) as resp:
                 result = await resp.text(encoding="utf-8")
 
+            account = None

```

Comment:
This needs to be before line 101 otherwise line 296 will fail with account is not defined.

Reply:
🤦🏻 fixed